src/libs/runtime-utils.js

```diff
@@ -65,6 +65,7 @@ export const defaultComputeRegion = 'US-CENTRAL1'
 
 export const defaultAutopauseThreshold = 30
 export const autopauseDisabledValue = 0
+
 // Leonardo considers autopause disabled when the threshold is set to 0
```

Comment:
Since you touched this line, I judge the comment belongs above the `const`. Feel free to revert if unintentional.
```suggestion
// Leonardo considers autopause disabled when the threshold is set to 0
export const autopauseDisabledValue = 0
```